### ISSUE TITLE
Search Files By Name Should Use Highlighted Text

### ISSUE THREAD

**Issue by machineghost** [microsoft/vscode#194573]
This has already been reported (and closed) as both https://github.com/microsoft/vscode/issues/59957 and https://github.com/microsoft/vscode/issues/59957.

This would be a great convenience: please stop closing it and just implement it :)

**Comment by nikhilkarve** [comment]
Can I take this up? I want to start contributing however possible, this would be my first issue, but I can give it a try.

**Comment by TylerLeonhardt** [comment]
@machineghost you linked the same issue twice... was there another one you're referring to?

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by machineghost** [comment]
> @machineghost you linked the same issue twice... was there another one you're referring to?

Yes, #97911 (I edited my original post).

**Comment by TylerLeonhardt** [comment]
@nikhilkarve you're welcome to try and take this on. Unfortunately, I can't help a ton as I do have a lot on my plate at the moment.

You can see what the new quickTextSearch is doing here:
https://github.com/microsoft/vscode/blob/43a2ba60de9d688785de36dd78b987817a07dd6c/src/vs/workbench/contrib/search/browser/searchActionsTextQuickAccess.ts#L45

and follow that kind of pattern for implementing it. You'll want to add this to here:
https://github.com/microsoft/vscode/blob/43a2ba60de9d688785de36dd78b987817a07dd6c/src/vs/workbench/browser/actions/quickAccessActions.ts#L121-L152

**Comment by nikhilkarve** [comment]
Sure, @TylerLeonhardt . I would love to try. And it's fine, if you don't mind I can post the questions here if I get any, you can answer anytime you are available. Is that okay?

**Comment by nikhilkarve** [comment]
Hello @TylerLeonhardt . I think I have implemented what is required. Attaching a small screen recording. Let me know if this is what's needed.

https://github.com/microsoft/vscode/assets/46160625/4e96e15e-e0e3-46d7-b2b8-c3fb55af54f0

Also, a quick question. As you suggested I made the changes to [vscode/src/vs/workbench/browser/actions/quickAccessActions.ts](https://github.com/microsoft/vscode/blob/43a2ba60de9d688785de36dd78b987817a07dd6c/src/vs/workbench/browser/actions/quickAccessActions.ts#L121-L152)

One thing I did is, I imported the following.
import { getSelectionTextFromEditor } from 'vs/workbench/contrib/search/browser/searchView';

But I am getting the error while committing saying the import is not allowed. I am not sure how to use that function here or shall I rewrite it in the browser folder? If you can just point me on how imports should be managed, I will work on that.

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by machineghost** [comment]
Who cares about votes?  It has a fix ready.

**Comment by nikhilkarve** [comment]
Hey @TylerLeonhardt , I just checked that it was moved to backlog. Let me know if there is anything needed to be done my side. Would try to wrap it up quickly. Thanks:)

**Comment by nikhilkarve** [comment]
@TylerLeonhardt @machineghost @andreamah 
Hello all!
How are y'all doing? It's been around a year.. are we planning to merge it? I can create a new PR if needed? Shall I go ahead and do it or are we planning to wait for some time?

**Comment by nikhilkarve** [comment]
Hi @TylerLeonhardt 
Are we waiting on anything?
